src/views/Home/Home.tsx

```diff
@@ -86,6 +88,12 @@ const CTACards = styled(BaseLayout)`
 
 const Home: React.FC = () => {
   const { t } = useTranslation()
+  const dispatch = useAppDispatch()
+
+  // Fetch farm data once to get the max APR
+  useEffect(() => {
+    dispatch(fetchFarmsPublicDataAsync(nonArchivedFarms.map((nonArchivedFarm) => nonArchivedFarm.pid)))
```

Comment:
Could further decrease amount of calls if we'd fetch only LIVE farms. NonArchicved includes finished too which is a bit useless.

Reply:
Ah, I didn't even think of that. Will update thanks!